From dd78e376759facfb07de8d5e5c6ff0a0d897cc41 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Mon, 25 Jan 2016 13:23:22 +0000
Subject: [PATCH] Show the menu label by default

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 panel-plugin/settings.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panel-plugin/settings.cpp b/panel-plugin/settings.cpp
index 71482c1..a33f0a6 100644
--- a/panel-plugin/settings.cpp
+++ b/panel-plugin/settings.cpp
@@ -84,7 +84,7 @@ Settings::Settings() :
 	m_modified(false),
 
 	button_icon_name("xfce4-whiskermenu"),
-	button_title_visible(false),
+	button_title_visible(true),
 	button_icon_visible(true),
 	button_single_row(false),
 
-- 
2.7.0

